scripts/get_nlvr_logical_forms.py

```diff
@@ -26,7 +26,7 @@ def process_data(input_file: str,
                  max_path_length: int,
                  max_num_logical_forms: int) -> None:
```

Comment:
You should note somewhere that this is the max number logical forms for _both_ correct and incorrect denotations.

Oh, I see that you did that in the argument parser below; having it here in the code would be nice, too.

Reply:
Done.